datahub-web-react/src/app/document/hooks/useSearchDocuments.ts

```diff
@@ -13,9 +13,20 @@ export interface SearchDocumentsInput {
     count?: number;
     fetchPolicy?: 'cache-first' | 'cache-and-network' | 'network-only';
     includeParentDocuments?: boolean;
+    /**
```

Comment:
TODO: Clean this up. It feels like we should just require 

sourceType or null.